members/CusSd2ecf4d21645569623.yaml

```diff
@@ -0,0 +1,119 @@
+bioguide: CusSd2ecf4d21645569623
```

Comment:
this looks like the same file as the one in https://github.com/NGPVAN/contact-congress/pull/1365/files, do we need them in both PRs?

Reply:
nope, i think this was a mistake with branching. I've asked Kat to redo it